### PR TITLE
Update Batch Management Swagger docs

### PR DESCRIPTION
**Pull request by matthchr**
This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/712)

### CODE REVIEW

**Review by BryanLa — COMMENTED** [review]
The description string changes/additions look good. Others will need to review the related schema changes/additions.

File: arm-batch/2015-12-01/swagger/BatchManagement.json [hunk 1]
```diff
@@ -79,14 +79,30 @@
...
-            "description": "",
+            "description": "The operation was successful. The response contains the Batch account entity.",
             "schema": {
               "$ref": "#/definitions/BatchAccount"
             }
+          },
+          "202": {
+            "description": "The operation will be completed asynchronously.",
+            "headers": {
+              "Location": {
```
Comment (amarzavery) [inline arm-batch/2015-12-01/swagger/BatchManagement.json]: should the location be all lowercase?

File: arm-batch/2015-12-01/swagger/BatchManagement.json [hunk 2]
```diff
@@ -79,14 +79,30 @@
...
+                "description": "The URL of the resource used to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "Suggested delay to check the status of the asynchronous operation. The value is an integer that represents the seconds.",
+                "type": "string"
```
Comment (amarzavery) [inline arm-batch/2015-12-01/swagger/BatchManagement.json]: The description says integer and the defined type is string

File: arm-batch/2015-12-01/swagger/BatchManagement.json [hunk 3]
```diff
@@ -174,14 +196,27 @@
...
```
Comment (amarzavery) [inline arm-batch/2015-12-01/swagger/BatchManagement.json]: same as above.

File: arm-batch/2015-12-01/swagger/BatchManagement.json [hunk 4]
```diff
@@ -174,14 +196,27 @@
...
-            "description": ""
+          "200": {
+            "description": "The operation was successful."
           },
-          "204": {
-            "description": ""
+          "202": {
+            "description": "The operation will be completed asynchronously.",
+            "headers": {
+              "Location": {
```
Comment (amarzavery) [inline arm-batch/2015-12-01/swagger/BatchManagement.json]: same as above

File: arm-batch/2015-12-01/swagger/BatchManagement.json (hunk 1 above)
Comment (matthchr) [inline arm-batch/2015-12-01/swagger/BatchManagement.json]: According to our service developers no, "Location" is supposed to be uppercase

**Review by amarzavery — CHANGES_REQUESTED** [review]

**Comment by matthchr** [comment]
@amarzavery Fixed the issues with `Retry-After`

**Review by amarzavery — APPROVED** [review]

**Comment by amarzavery** [comment]
Should be merged once the CI passes

**Pull request merged**